Resources/Prototypes/_NF/Entities/Objects/Tools/umbrella.yml

```
@@ -0,0 +1,68 @@
+﻿- type: entity
+  name: umbrella
+  parent: BaseItem
+  id: FlashlightUmbrella
+  description: It lights the way to freedom.
```

Inherited from the lantern - suggested rewrite.
```suggestion
  description: Keeps you dry AND stylish.
```